ir/src/hir/builder.rs

```diff
@@ -505,6 +504,7 @@ pub trait InstBuilder<'f>: InstBuilderBase<'f> {
     binary_int_op!(rotr, Opcode::Rotr);
     unary_numeric_op!(neg, Opcode::Neg);
     unary_numeric_op!(inv, Opcode::Inv);
+    binary_int_op!(pow2, Opcode::Pow2);
```

Comment:
Did you mean to make this `unary_int_op!(pow2, Opcode::Pow2)`?

Reply:
Ah, yes. Moving the code without changing it... Give me a minute.